.jenkins/library/vars/helpers.groovy

```diff
@@ -477,3 +481,37 @@ def releaseInstall(String release_version = null, String oe_package = "open-encl
         """
     }
 }
+
+/**
+ * Get today's date in YYYYMMDD format
+ *
+ * @param delimiter  Optional argument to place a delimiter between YYYY, MM, and DD.
+ */
+def get_date(String delimiter = "") {
+    return (
+        LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy")) +
+        delimiter +
+        LocalDateTime.now().format(DateTimeFormatter.ofPattern("MM")) +
+        delimiter +
+        LocalDateTime.now().format(DateTimeFormatter.ofPattern("dd"))
+    )
+}
+
+/* Returns Azure Image URNs for OS type
+ *
+ * @param os_type  The OS distribution (Currently only "ubuntu")
+ * @param os_version The OS distribution version ("18.04", "20.04")
+ */
+ def getAzureImageUrn(String os_type, String os_version) {
+    if (os_type.toLowerCase() == 'ubuntu') {
```

Comment:
It's a stylistic thing, but it might be easier to read if the error cases were caught at the start:
```
if (os_type != 'ubuntu' || os_version != 20.04 || os_version != 18.04) {
  // error
}
//18.04 and 20.04 logic
```

Reply:
Yeah, I think it might be easier to read. Will make this change, thanks!